### PR TITLE
ffi: aggregate `RoomMember` fields early instead of requiring calling getters

### PR DESCRIPTION
**Pull request by timokoesters**
See https://github.com/matrix-org/matrix-rust-sdk/pull/3074 and https://github.com/matrix-org/matrix-rust-sdk/pull/3101 for previous versions. This PR is based on !3074 with minor cleanup.
The corresponding element android PR is here: https://github.com/element-hq/element-x-android/pull/2322

- [ ] Public API changes documented in changelogs (optional)

### CODE REVIEW

File: bindings/matrix-sdk-ffi/src/room_member.rs [hunk 1]
```diff
@@ -154,3 +154,16 @@ pub fn suggested_role_for_power_level(power_level: i64) -> RoomMemberRole {
     // It's not possible to expose the constructor on the Enum through Uniffi ☹️
     RoomMemberRole::suggested_role_for_power_level(power_level)
 }
+
+#[derive(uniffi::Record)]
+pub struct ExportedRoomMember {
```
Comment (bnjbvr) [inline bindings/matrix-sdk-ffi/src/room_member.rs:159]: Could we replace `RoomMember` with this `ExportedRoomMember`, or merge them one way or another?

**Review by bnjbvr — COMMENTED** [review]
Nice! In addition to my question below, can you run formatting please? Thanks!

File: bindings/matrix-sdk-ffi/src/room_member.rs (hunk 1 above)
Comment (bnjbvr) [reply bindings/matrix-sdk-ffi/src/room_member.rs:49]: Thanks for merging them; can you rename `ExportedRoomMember` to `RoomMember`, then?

**Review by bnjbvr — APPROVED** [review]
I'll fix the last comment, thanks.

**Comment by jmartinesp** [comment]
Codecov is (unsurprisingly) stuck

**Pull request merged**